infrastructure/api-configuration/nginx_config.conf

```diff
@@ -62,50 +62,6 @@ http {
             proxy_pass http://app;
             proxy_read_timeout 30s;
             proxy_send_timeout 30s;
-
-            # Uncomment this to add CORS at the nginx level.
-            # This Django app uses `corsheaders` now, so duplicating here causes problems.
-            # if ($request_method = 'OPTIONS') {
-            #    add_header 'Access-Control-Allow-Origin' '*';
-            #    add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, OPTIONS';
-            #    #
-            #    # Custom headers and headers various browsers *should* be OK with but aren't
-            #    #
-            #    add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
-            #    #
-            #    # Tell client that this pre-flight info is valid for 20 days
-            #    #
-            #    add_header 'Access-Control-Max-Age' 1728000;
-            #    add_header 'Content-Type' 'text/plain; charset=utf-8';
-            #    add_header 'Content-Length' 0;
-            #    return 204;
-            # }
-            # if ($request_method = 'POST') {
-            #     add_header 'Access-Control-Allow-Origin' '*';
-            #     add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, OPTIONS';
-            #     add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
-            #     add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
-            # }
-            # if ($request_method = 'GET') {
-            #     add_header 'Access-Control-Allow-Origin' '*';
-            #     add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, OPTIONS';
-            #     add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
-            #     add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
-            # }
-            # if ($request_method = 'PUT') {
-            #     add_header 'Access-Control-Allow-Origin' '*';
-            #     add_header 'Access-Control-Allow-Methods' 'GET, POST, PUT, OPTIONS';
-            #     add_header 'Access-Control-Allow-Headers' 'DNT,User-Agent,X-Requested-With,If-Modified-Since,Cache-Control,Content-Type,Range';
-            #     add_header 'Access-Control-Expose-Headers' 'Content-Length,Content-Range';
-            # }
-        }
-
-        #error_page  404              /404.html;
```

Comment:
did you want to add a custom nginx error page to handle the various errors?

Reply:
We don't have a custom page we want to show right? As far as I can tell this behaves well when there's a 500 or 404.